zeebe/zb-db/src/main/java/io/camunda/zeebe/db/ZeebeDb.java

```diff
@@ -66,4 +66,6 @@ ColumnFamily<KeyType, ValueType> createColumnFamily(
   boolean isEmpty(ColumnFamilyType column, TransactionContext context);
 
   StatefulMeterRegistry getMeterRegistry();
```

Comment:
:wrench: if you could please fix my mistake, and add docs to describe when to use this registry, and its lifecycle.